src/components/CollaborationBar/Collaborators/Collaborators.tsx

```diff
@@ -36,21 +37,37 @@ import {
   ToolbarAvatarGroup,
   ToolbarAvatarMoreButton,
 } from '../../common/Toolbar';
+import { ToolbarAvatarPresenter } from '../../common/Toolbar/ToolbarAvatarPresenter';
 import { ellipsis } from '../../common/ellipsis';
 import { orderMembersByState } from './orderMembersByState';
 
 export function Collaborators() {
   const { t } = useTranslation();
+  const { state } = usePresentationMode();
   const ownUserId = useWidgetApi().widgetParameters.userId;
+  const isViewingPresentation = state.type === 'presenting' || 'presentation';
+  const presentingUser =
+    state.type === 'presentation'
+      ? state.presenterUserId
+      : state.type === 'presenting'
+        ? ownUserId
+        : undefined;
 
   if (!ownUserId) {
     throw new Error('Unknown user id');
   }
 
   const activeMembers = useActiveWhiteboardMembers();
-  const orderedMembers = orderMembersByState(activeMembers, ownUserId);
+  const orderedMembers = isViewingPresentation
+    ? orderMembersByState(activeMembers, ownUserId).filter(
+        (item) => item.userId !== presentingUser,
+      )
+    : orderMembersByState(activeMembers, ownUserId);
+
   const maxAvatars = 5;
-  const furtherMembers = orderedMembers.slice(maxAvatars);
+  const furtherMembers = orderMembersByState(activeMembers, ownUserId).slice(
+    maxAvatars,
+  );
```

Comment:
This removes presenting user from the members but there is no benefit to do that because it makes `furtherMembers` evaluation necessary and later in this file presenter toolbar avatar is added explicitly (with it's own issue described later). 

Instead we could modify `orderMembersByState` to prepend presenting user and return it before own user if presenting user is not an own user (if they are the same just return one entry).